### DISCUSSION TITLE
How to filter on multiple conditions containing wildcards?

### DISCUSSION THREAD

**Discussion by BristleBeard** [kannibalox/pyrosimple#16] [Q&A] [answered]
This may be a simple issue of syntax, but how would I achieve a filter on something like:
`rtcontrol message=*nregistered*,*not?registered*,*not?authorized*` 

I have tried several ways but no success so far.

**Comment by kannibalox** [comment, marked as answer]
The easiest way right now is to convert it to a regex:
```
rtcontrol message='/(un|not.)registered|not.authorized/i'
```
There were some bugs with regards to prefiltering that I just fixed up, so if you had tried it earlier with fast querying enabled it might have behaved weirdly.
